### DISCUSSION TITLE
Add system total network traffic count and TCP connections count

### DISCUSSION THREAD

**Discussion by Muska-Ami** [henrygd/beszel#165] [Ideas]
Maybe we need a total traffic count and TCP connections count data like:

![70d2af2239cc39fbcac34f7ba545db9d](https://github.com/user-attachments/assets/87df3f13-a4fe-41ae-bdb4-4cf05326d3d1)

Note: The TCP connections also can be a chart.(I think maybe it would be more intuitive as a chart? :D)

**Comment by scriptzteam** [comment]
+1
but also UDP would be great

**Comment by ghost** [comment]
I love the idea of having bandwidth usage on the dashboard, it would be really good if it would calculate bandwidth usage based upon the current timeframe (1h, 12h , 24h etc)
